src/components/JournalPage/index.jsx

```diff
@@ -22,9 +22,11 @@ import './JournalPage.scss';
 
 
 class JournalPage extends React.Component {
-  componentDidMount() {
+  constructor(props) {
+    super(props);
```

Comment:
why was this change needed?

Reply:
The react lifecycle goes:
1) constructor()
2) static getDerivedStateFromProps()
3) render()
4) componentDidMount()

So we were actually requesting the page after we were asking the page to render. This just makes us wait less for the page to render correctly.